source/features/status-subscription.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'dom-chef';
+import * as pageDetect from 'github-url-detection';
+import {BellIcon, BellSlashIcon, IssueReopenedIcon} from '@primer/octicons-react';
+
+import features from '../feature-manager.js';
+import observe from '../helpers/selector-observer.js';
+
+const disableAttrs = {
+	'aria-selected': true,
+	className: 'selected',
+	style: {pointerEvents: 'none'},
+} as const satisfies React.HTMLAttributes<HTMLButtonElement>;
+
+function SubButton(): JSX.Element {
+	return (
+		<button
+			data-disable-with
+			name="id"
+			type="submit"
+			className="btn btn-sm flex-grow-1 BtnGroup-item tooltipped tooltipped-sw"
+		/>
+	);
+}
+
+function getCurrentStatus(subscriptionButton: HTMLButtonElement): 'none' | 'all' | 'status' {
+	const reason = subscriptionButton
+		.closest('.thread-subscription-status')!
+		.querySelector('.reason')!
+		.textContent!;
+
+	// You’re receiving notifications because you chose custom settings for this thread.
+	if (reason.includes('custom settings')) {
+		return 'status';
```

Comment:
The "Status" button will be highlighted even if the just one event was selected, there's no "partially-selected" status. Not something I'm interested in fixing, just click "None" and then "Status" to get the full status subscription. "Customize" shows the correct status anyway.